tests/test_prod_info.py

```diff
@@ -205,6 +205,42 @@ def test_deserialize_update_prod_info(self):
         prod_info = ProductInformation(prod_name = "ert", new_qty = -1, used_qty = -1, open_boxed_qty = -1, restock_level = -2, restock_amt = -1)
         self.assertRaises(DataValidationError, prod_info.deserialize_update, data)
 
+    def test_restock_exception(self):
+        """ 
+        Test restock() when property is not initialized.
+        """
+        # Missing 'restock_level'
+        prod_info = ProductInformation(prod_id=1, prod_name='Storm Trooper', \
+                new_qty=20, used_qty=30, open_boxed_qty=40, \
+                restock_amt=100)
+        self.assertRaises(DataValidationError, prod_info.restock)
+
+    def test_restock(self):
+        """ Test restocking functionality. """
+        prod_info = ProductInformation(prod_id=1, prod_name='Storm Trooper', \
+                new_qty=20, used_qty=30, open_boxed_qty=40, \
+                restock_level=1000, restock_amt=100)
+
+        # Training a clone army for the Republic.
```

Comment:
What does this mean?

Reply:
Lol this is a mime related to the Star Wars series (noting that the prod_name is 'Storm Trooper')!